### ISSUE TITLE
Newly added text element does not receive keyboard focus automatically

### ISSUE THREAD

**Issue by barklund** [swissspidy/web-stories-wp#9781]
## Bug Description

When adding a text element, it does not automatically receive keyboard focus

## Expected Behaviour

A newly added text element should have keyboard focus

## Steps to Reproduce

* Add a new text element using the + in the element library tab bar
* Press enter key to edit text
* Observe that the text element did in fact not go into edit mode, because it does not have keyboard focus
